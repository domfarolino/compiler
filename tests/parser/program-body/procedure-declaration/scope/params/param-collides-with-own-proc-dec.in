program dominic is
  procedure function1(integer function1 in)
  begin
  end procedure;

begin
end program.
