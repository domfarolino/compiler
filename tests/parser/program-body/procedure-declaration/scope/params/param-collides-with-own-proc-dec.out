[1;31mLine 3: Cannot add parameter 'function1' to the current scope, symbol name already exists
Line 3: Error parsing program body
[0m